galpy/potential_src/potential_c_ext/SpiralArmsPotential.c

```diff
@@ -0,0 +1,763 @@
+#include <math.h>
+#include <galpy_potentials.h>
+
+double gam(double R, double phi, double N, double phi_ref, double r_ref, double tan_alpha);
+
+double dgam_dR(double R, double N, double tan_alpha);
+
+double K(double R, double n, double N, double sin_alpha);
+
+double B(double R, double H, double n, double N, double sin_alpha);
+
+double D(double R, double H, double n, double N, double sin_alpha);
+
+double dK_dR(double R, double n, double N, double sin_alpha);
+
+double dB_dR(double R, double H, double n, double N, double sin_alpha);
+
+double dD_dR(double R, double H, double n, double N, double sin_alpha);
+
+double SpiralArmsPotentialEval(double R, double z, double phi, double t,
```

Comment:
It's great that you've added this function, but currently it will not get used by ``galpy`` (because evaluating a potential in C is only done for action-angle calculations, which we don't support for complicated potentials such as this one). You can exclude this code black from the report that checks whether the line is covered by the tests by adding a line before this definition that says
```
//LCOV_EXCL_START
```
and another line after it that I will list below.